### PR TITLE
Different way of using routes

### PR DESCRIPTION
**Pull request by taylortrimble**
The diff speaks for itself

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 SECRET_KEY = '6C863A81-5C37-47BE-9D7D-362F073F7BE7'
 
 CONTACT_EMAIL = 'officers@teslaworks.net'
-PRESIDENT_EMAIL = 'president@teslaworks.net'
```
Comment (mplewis) [inline config.py:5]: unrelated change

File: server_dev.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def start_project():
     return redirect(url_for('index'))
 
 @app.route('/<dynamic>')
-def project(dynamic):
+def dynamic(dynamic):
```
Comment (mplewis) [inline server_dev.py:67]: Is this related?

**Comment by mplewis** [comment]
I like it. :snail:

File: config.py (hunk 1 above)
Comment (taylortrimble) [inline config.py:5]: With the new `/contact` redirect, we officially won't use the president's email anymore.

File: server_dev.py (hunk 2 above)
Comment (taylortrimble) [inline server_dev.py:67]: Kind of. `dynamic` services both projects and redirects now.
